packages/angular/src/table/head/table-head-cell.scss

```diff
@@ -0,0 +1,51 @@
+@import '../../../../styles/scss/globals/vars';
```

Comment:
Same style import comment as the other PR, should be pointing to `../../vendor/@ai-apps/styles/scss`

Reply:
There's also quite a few styles here, is there anything here we can just pull up into the main PAL styles?